web/src/containers/Trade/components/TradeBlock.js

```diff
@@ -25,9 +28,25 @@ const TradeBlock = ({
 	tool,
 	toggleTool,
 	titleClassName = '',
+	ticker,
 }) => {
 	const pairs = pair ? pair.split('-').map((curr) => curr.toUpperCase()) : [];
 	const { icon_id } = pairData;
+	const [stats, setStats] = useState({
+		maxPrice: 0,
+		lowPrice: 0,
+		summary: 0,
+	});
+
+	useEffect(() => {
```

Comment:
Sending requests here will send 8 requests simultaneously since all trade tools are wrapped within the trade block. We should lift this logic up (both data and rendering) to the trade index. Then pass that as a prop to the trade block.